src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,19 @@
+.stars {
+  display: flex;
+
+  &__star {
+    height: 16px;
+    width: 16px;
+    margin-right: 4px;
+    background: url(../images/star.svg) no-repeat center;
+  }
+  
+}
+
+.stars--1 :nth-child(1),
+.stars--2 :nth-child(-n +2),
+.stars--3 :nth-child(-n +3),
+.stars--4 :nth-child(-n +4),
+.stars--5 :nth-child(n) {
```

Comment:
paragraph 12 in checklist 
[SASS] - Try using for loops for the stars selector.

Reply:
Agree